pygazebo/pygazebo.cc

```diff
@@ -264,6 +265,24 @@ class Agent : public Model {
     }
   }
 
+  bool TakeActionPDController(const std::map<std::string, double>& ctrl_target) {
+    // Create a PID controller
+    // The parameters suits for revolute/continuous joint, from error of radian to force
+    auto pid = gazebo::common::PID(0.02, 0, 0.01);
+    pid.SetCmdMax(10.0);
+    pid.SetCmdMin(-10.0);
+    auto controller = model_->GetJointController();
+    for (const auto& name2target : ctrl_target) {
+      controller->SetVelocityPID(name2target.first, pid);
+      bool ret = controller->SetVelocityTarget(name2target.first, name2target.second);
```

Comment:
We should set PID separately in a different function and expose that to python SDK because the PID parameters might be different for different joints.

Reply:
Sure, we should. Will update it later.